knative/knative-serving-install/base/config-map.yaml

```diff
@@ -658,7 +658,7 @@ data:
     # {{ingress_namespace}}.svc.cluster.local"`. The {{gateway_namespace}}
     # is optional; when it is omitted, the system will search for
     # the gateway in the serving system namespace `knative-serving`
-    gateway.kubeflow.kubeflow-gateway: "istio-ingressgateway.istio-system.svc.cluster.local"
+    gateway.kubeflow.kubeflow-gateway: "kubeflow-gateway.kubeflow"
```

Comment:
why we need to change this? it should be the istio ingress gateway service name

Reply:
Reverted. Thanks.